watchout-production.js

```diff
@@ -120,36 +516,38 @@ instance.prototype.destroy = function() {
 		self.socket = undefined;
 	}
 
+	self.stopPolling();
+
 	debug("destroy", self.id);
 };
 
 instance.prototype.actions = function(system) {
 	var self = this;
-	self.setActions({
+	let timelineOption = {
+		type: 'textinput',
+		label: 'timeline (optional)',
+		id: 'timeline',
+		default: ''
+	}
+	if(self.config.feedback == 'simple' || self.config.feedback == 'advanced') {
+		timelineOption = {
```

Comment:
It would be good for this to use the `allowCustom: true` capability here so options can be typed in manually still.

Reply:
That's a great option! Added, didn't know about it since it's not in the wiki, thanks.